zerver/lib/actions.py

```diff
@@ -5900,3 +5900,61 @@ def do_delete_realm_export(user_profile: UserProfile, export: RealmAuditLog) ->
     export.extra_data = ujson.dumps(export_data)
     export.save(update_fields=['extra_data'])
     notify_realm_export(user_profile)
+
+def get_topic_messages(
+    user_profile: UserProfile,
+    stream: Stream,
+    topic_name: str
+) -> List[Message]:
```

Comment:
This line-wrapping doesn't match zulip's style.  Please look at nearby code and follow our conventions.